portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+
+/** Models a task. It can be used for both creation and import flow. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Duration durationMinute;
+  private final TaskPriority priority;
+
+  // TODO(raulcruise): Add an ID field if necessary.
+
+  // The constructor will make sure that all necessary parameters are passed in,
+  // and populate each class variable appropriately.
+  public Task(String name, String description, Duration durationMinute, Integer priority) {
```

Comment:
I would take TaskPriority instead of Integer. Otherwise, caller can pass a value like -100, which I assume is invalid.

And as I mentioned above, we can rename durationMinute to duration.

Reply:
Done.